bin/pycbc_banksim

```diff
@@ -374,7 +374,7 @@ with ctx:
                                       template_sample_rate,
                                       filter_N, options.filter_sample_rate)
 
-            h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
+                h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
```

Comment:
Note that I don't actually have a strong opinion on the usefulness of `--filter-high-freq-cutoff-isco-factor`. But if we *do* decide to break it in favor of the optimization, then let's just remove the whole option and related code.